### ISSUE TITLE
Incompatible with `bytes` version `1.2` and above

### ISSUE THREAD

**Issue by theRookieCoder** [XAMPPRocky/octocrab#235]
Because of the dependency on `hyperx` (whose repository is stale/inactive), the `size` dependency cannot be upgraded on any project that uses `octocrab`.

**Comment by XAMPPRocky** [comment]
Thank you for your issue! As noted in #110 I'm not interested in replacing `hyperx` without a viable alternative, so this issue is blocked on `headers` or an equivalent crate providing that alternative. See https://github.com/hyperium/headers/pull/113 for the latest attempt to get this merged into `headers`. For now I'm going to close this as a duplicate of #110

**Comment by theRookieCoder** [comment]
Great, tahank you so much for creating and maintaining this project!

**Comment by theRookieCoder** [comment]
Just a little update, because of hyperx `url` is also bound to `2.2`

**Comment by theRookieCoder** [comment]
Thanks for fixing this issue! The 2 dependency woes I had with `url` and `bytes` have been fixed now!
